src/main/java/ca/on/oicr/gsi/dimsum/service/CaseService.java

```diff
@@ -158,6 +159,13 @@ public Set<String> getMatchingRunNames(String prefix) {
         .collect(Collectors.toSet());
   }
 
+  public Set<String> getMatchingTestNames(String prefix) {
+    Set<Test> tests = caseData.getCases().stream().flatMap(kase -> kase.getTests().stream())
+        .filter(s -> s.getName().toLowerCase().startsWith(prefix.toLowerCase()))
+        .collect(Collectors.toSet());
+    return tests.stream().map(test -> test.getName()).collect(Collectors.toSet());
+  }
```

Comment:
This could potentially get slow as more and more cases are added to Dimsum, and it gets called rapidly as someone is typing. Collect the test names once in `CaseLoader` and store in `CaseData` instead